roles/openshift_health_checker/openshift_checks/etcd_traffic.py

```diff
@@ -0,0 +1,59 @@
+"""Module that scans journalctl for messages caused as a symptom of increased etcd traffic"""
+
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EtcdTraffic(OpenShiftCheck):
+    """Check if host is being affected by an increase in etcd traffic."""
+
+    name = "etcd_traffic"
+    tags = ["health", "etcd"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        """Skip hosts that do not have etcd or masters in their group names."""
+        master_count = get_var(task_vars, "openshift", "master", "master_count", default=1)
+        group_names = get_var(task_vars, "group_names", default=[])
+        active = "masters" in group_names
+        if int(master_count) > 1:
+            active = "etcd" in group_names
```

Comment:
We want this to only be active if `masters` is in `group_names` on a single master setup. If there is more than one master, we only want this to be active if `etcd` is in `group_names`. Reason for this is because on multimaster clusters, etcd tends to be on a host of its own if I recall correctly. @sosiouxme